### PR TITLE
Feat/fetch upstream

### PR DESCRIPTION
**Pull request by jbust23**

### CODE REVIEW

**Review by Mejiro-McQueen — APPROVED** [review]

File: ait/dsn/sle/common.py [hunk 1]
```diff
@@ -278,25 +278,37 @@ def connect(self):
         Initialize TCP connection with DSN and send context message
         to configure communication.
         '''
-        self._socket = gevent.socket.socket(socket.AF_INET, socket.SOCK_STREAM)
```
Comment (Mejiro-McQueen) [inline ait/dsn/sle/common.py:281]: This commit is a major change from what we had in DTF-21. 
We'll need to do some thorough testing.

Comment (cSchubes) [reply ait/dsn/sle/common.py:281]: @Mejiro-McQueen can you comment further on the changes you see, so we have a record of it? It seems like at least your hotfix from DTF-21 (lines 165-167) is intact, is that correct?

File: doc/source/conf.py [hunk 2]
```diff
@@ -58,9 +58,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = u'2.0.0'
+version = u'2.0.1-rc1'
```
Comment (cSchubes) [inline doc/source/conf.py:61]: Should this match our version from `setup.py`?

File: ait/dsn/sle/common.py (hunk 1 above)
Comment (Mejiro-McQueen) [reply ait/dsn/sle/common.py:281]: The DTF-21 hotfix is correct (165-167) as well as import gevent.select on line 60.
Original: https://github.com/sunrise-cubesat/AIT-DSN/pull/8/commits/c527bea68f8f8b9c86188263b48edf9c00b4841e
Fixup: https://github.com/sunrise-cubesat/AIT-DSN/pull/8/commits/0d20f8344263d60f7c2fa64cfbb171a515f922d4

Everything are from different commits and is what I'm referring to.
Major Change: https://github.com/sunrise-cubesat/AIT-DSN/pull/8/commits/8efc76f9a06d7af1ebb99402984b028236f8cc35
Minor Change (Don't worry about it): https://github.com/sunrise-cubesat/AIT-DSN/pull/8/commits/017854200dc51929dff7c7662bc2f7f52dd8eb34

Comment (cSchubes) [reply ait/dsn/sle/common.py:281]: understood, gonna link this issue to make sure we know to test this:

https://jira.jpl.nasa.gov/browse/SRGDS-83

**Review by cSchubes — APPROVED** [review]
approving so merge can happen despite question about version number

File: doc/source/conf.py (hunk 2 above)
Comment (jbust23) [reply doc/source/conf.py:61]: Similar to AIT-Core I didnt bother updating the docs but I can address that at a later time if needed

**Pull request merged**